### DISCUSSION TITLE
Running multiple projects with one installation

### DISCUSSION THREAD

**Discussion by truefantomas** [codeigniter4/shield#302] [Q&A]
WORKS: shield and settings and framework
appstarter/
        ...app/
        ...public/
        ...vendor/codeigniter4/(framework, settings and shield)

NOT WORK: settings and shield
vendor/codeigniter4/(settings and shield)
appstarter/
        ...app/
        ...public/

WORKS: shield and settings and framework
vendor/codeigniter4/
appstarter/
        ...app/
        ...public/
        ...vendor/codeigniter4 (folder copy)

I don't think it should be like this

**Comment by kenjis** [comment]
The `appstarter` is the root folder of the project.
Why do you need to move the `vendor` out of the project?

**Reply by truefantomas** [reply]
app/Config/Paths ROOTPATH ​​allows you to use codeigniter4 for multiple projects at the same time.  This is even mentioned somewhere in the documentation, if I find it, I will share the link.  p.s on the main site codeigniter.com the http:// to https:// redirect does not work, this can create unwanted links in the search engine, which can lead to duplicate pages in search ... you probably know yourself ;-)

**Reply by kenjis** [reply]
See https://codeigniter4.github.io/CodeIgniter4/general/managing_apps.html#running-multiple-applications-with-one-codeigniter-installation

**Reply by truefantomas** [reply]
yes, I was talking about her.  this is how my projects are set up.  lonniezzel gave the answer, hope it works.  Thanks.

**Comment by lonnieezell** [comment]
Change the value for COMPOSER_PATH in `app/Config/Constants.php` to point to your desired `vendor` location.

```php
defined('COMPOSER_PATH') || define('COMPOSER_PATH', ROOTPATH . 'vendor/autoload.php');
```

**Reply by truefantomas** [reply]
Thanks!!!

**Reply by MGatner** [reply]
This doesn't technically have anything to do with Shield but I'm updating the title so it might be helpful for others with the same situation.
